src/PrestaShopBundle/Controller/Admin/Improve/Design/ThemeController.php

```diff
@@ -599,6 +601,12 @@ private function getLogoUploadErrorMessages(DomainException $exception)
             [$availableLogoFormatsImploded]
         );
 
+        $mailAndInvoiceImageFormatError = $this->trans(
+            'Image format not recognized, allowed format(s) is(are): .%s',
```

Comment:
```suggestion
            'Image format not recognized, allowed formats are: %s',
```
It would be great if you could use the new string (included in a previous PR from Franck) as the current one is not really easy to translate in some languages.